File: 1157.py
n = int(input())
lista = []
vezes = 0
for i in range(0, n+1):
    if i != 0:
        if n % i == 0:
            lista += [i]
            vezes += 1
for l in range(0, vezes):
    print(f'{lista[l]}')

File: 1157.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let n = parseInt(lines.shift());
let lista = [];
let vezes = 0;
for(var i = 0; i < n+1; i++){
    if(i !== 0){
        if(n % i === 0){
            lista.push(i);
            vezes += 1;
        }
    }
}
for(var l = 0; l < vezes; l++){
    console.log(`${lista[l]}`);
}
